src/test/java/uk/gov/hmcts/reform/bailcaseapi/domain/handlers/postsubmit/DocumentsUploadedConfirmationTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.bailcaseapi.domain.handlers.postsubmit;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoSettings;
+import org.mockito.quality.Strictness;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.BailCase;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.CaseDetails;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.Event;
+import uk.gov.hmcts.reform.bailcaseapi.domain.entities.ccd.callback.Callback;
+
+@MockitoSettings(strictness = Strictness.LENIENT)
+public class DocumentsUploadedConfirmationTest {
+
+    @Mock private Callback<BailCase> callback;
+
+    @Mock private CaseDetails<BailCase> caseDetails;
+    private DocumentsUploadedConfirmation documentsUploadedConfirmation;
+
+    @BeforeEach
+    public void setUp() {
+        documentsUploadedConfirmation = new DocumentsUploadedConfirmation();
+        when(callback.getEvent()).thenReturn(Event.UPLOAD_DOCUMENTS);
+    }
+
+    @Test
+    void should_not_allow_null_args() {
+        assertThatThrownBy(() -> documentsUploadedConfirmation.handle(null))
+            .isExactlyInstanceOf(NullPointerException.class);
+    }
+
+    @Test
```

Comment:
Can you please add a test to check the confirmation body and header text.

Reply:
Added